datasets/us/trade_csl/us_trade_csl.yml

```diff
@@ -100,6 +114,277 @@ lookups:
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Ningbo Huawei Computer & Net Co., Ltd.
+          notes: "Affiliated Entity: Ningbo Huawei Computer & Net Co., Ltd."
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity: Shanghai Huawei Technologies Co., Ltd."
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Shanghai Huawei Technologies Co., Ltd.
+          notes: "Affiliated Entity: Shanghai Huawei Technologies Co., Ltd."
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity: Shenzhen Huawei New Technology Co., Ltd."
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Shenzhen Huawei New Technology Co., Ltd.
+          notes: "Affiliated Entity: Shenzhen Huawei New Technology Co., Ltd."
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity: Shenzhen Huawei Technology Service"
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Shenzhen Huawei Technology Service
+          notes: "Affiliated Entity: Shenzhen Huawei Technology Service"
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity: Shenzhen Huawei Technologies Software"
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Shenzhen Huawei Technologies Software
+          notes: "Affiliated Entity: Shenzhen Huawei Technologies Software"
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity: Zhejiang Huawei Communications Technology Co., Ltd."
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Zhejiang Huawei Communications Technology Co., Ltd.
+          notes: "Affiliated Entity: Zhejiang Huawei Communications Technology Co., Ltd."
+      - match: "Affiliated Supply and Consultancy Services"
+        properties:
+          name: Affiliated Supply and Consultancy Services
+      - match: "Huawei Technologies Co., Ltd. (Huawei).  Affiliated Entity:  Shenzhen Huawei Anjiexin Electricity Co., Ltd."
+        properties:
+          name: Huawei Technologies Co., Ltd. (Huawei)
+          related: Shenzhen Huawei Anjiexin Electricity Co., Ltd
+          notes: "Affiliated Entity:  Shenzhen Huawei Anjiexin Electricity Co., Ltd."
+      - match: "China Aerospace Science and Industry Corporation Second Academy.  Subordinate Institution -  Second Design Department"
+        properties:
+          name: China Aerospace Science and Industry Corporation Second Academy
+          related: Second Design Department
+          notes: "Subordinate Institution -  Second Design Department"
+      - match: "China Aerospace Science and Industry Corporation Second Academy.  Subordinate Institution - 23rd Research Institute"
```

Comment:
Hm, for these branches I feel it makes sense to keep around the full name or put the branch name in parentheses or with a comma (like for the Huawei departments above). I think "23rd Research Institute" on its own isn't really enough to make sense.